app/components/institution-page/campus-program-application-modal.ts

```diff
@@ -16,6 +18,8 @@ interface Signature {
 
 export default class CampusProgramApplicationModal extends Component<Signature> {
   @service declare authenticator: AuthenticatorService;
+  @tracked emailAddressToPrefill: string | undefined = undefined;
```

Comment:
Don't think we need two pieces of state here - `emailAddressToPrefill` can just always be `latestSavedGrantApplication?.normalizedEmailAddress`.